client/zeth/core/signing.py

```diff
@@ -11,11 +11,12 @@
 BN128
 """
 
+from __future__ import annotations
 from math import ceil
 from os import urandom
 from hashlib import sha256
 from py_ecc import bn128 as ec
-from typing import List, Tuple
+from typing import Dict, List, Tuple, Any
 
 FQ = ec.FQ
 G1 = Tuple[ec.FQ, ec.FQ]
```

Comment:
Out of curiosity, can't we set `SIGNATURE_PRIME` via `curve_order` for `py_ecc`? (https://github.com/ethereum/py_ecc/blob/master/py_ecc/bn128/bn128_curve.py#L21).

Reply:
I'm not familiar with th original code, but this sems to work.  Replaced.

Reply:
Perfect thanks. That's good to remove "magic numbers" as much as we can (thought these are not so magic, we know where they come from, but still ;) )